src/components/select/utils/is-navigation-key.spec.js

```diff
@@ -0,0 +1,30 @@
+import isNavigationKey from "./is-navigation-key";
+
+describe("isNavigationKey", () => {
+  describe("correct navigation keys", () => {
+    it.each(["ArrowDown", "ArrowUp", "Home", "End", "PageUp", "PageDown"])(
+      "%s is counted as a navigation key",
+      (key) => {
+        expect(isNavigationKey(key)).toBe(true);
+      }
+    );
+  });
+
+  describe("non-navigation keys", () => {
+    it.each(["a", "q", "z", "1", "0", " ", "Enter", "Tab", "Delete"])(
+      "%s is not counted as a navigation key",
+      (key) => {
+        expect(isNavigationKey(key)).toBe(false);
+      }
+    );
+  });
+
+  describe("left/right arrow keys are not counted as navigation keys as far as Select is concerned", () => {
```

Comment:
```suggestion
  describe("left/right arrow keys", () => {
```
No need for the extra description here, now the full test output will read:

> isNavigationKey > left/right arrow keys > ArrowLeft/ArrowRight is not counted as a navigation key